### DISCUSSION TITLE
Interest in UTM for MacAdmins Presentation?

### DISCUSSION THREAD

**Discussion by uurazzle** [utmapp/UTM#4217] [General]
This is Richard Glaser from the University of Utah. I head popular monthly MacAdmin meetings and manage multiple IT teams in our department and we actively contribute to the community with open source, presentations, and documentation.

I was wondering if you might be interested in doing a remote presentation on “Wed, Aug 17, 2022” or at a future meeting?

I was thinking that an update on “UTM for MacAdmins” or if you have a suggestion for another topic I would be interested in hearing about it.

The meetings are held on the 3rd Wednesday of each month at 11 AM MT.

If you aren’t familiar with our meetings or team, see the following links:

Apple Infrastructure Github Repositories
Apple Infrastructure Blog
University of Utah Meeting Archives

If you are interested or need additional information, please let me know.

If you are interested, I will need a brief description of your presentation, a bio and an email I can send the meeting/presentation invite and zoom meeting details. Please browse the existing issues and filter by 'enhancement' to make sure this has not already been suggested!

**Comment by osy** [comment]
I think this is a great idea. While I personally cannot attend, other contributors are welcome to do it.

**Reply by uurazzle** [reply]
Hi @osy:

The meetings are held on the 3rd Wednesday of each month at 11 AM MT. So, if you can't make one month other month's meetings might be an option? Fedor Korotkov the creator of Tart is currently presenting and would love to get a UTM presentation & viewpoint.
